<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart Medical Center</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Bureau/projet_image/mw.png</normaloff>../../Bureau/projet_image/mw.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>751</width>
       <height>331</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="tabSpacing">
      <number>6</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>223</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Ajouter Patient</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>369</width>
         <height>158</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Prenom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Age</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Sexe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Lieu de résidence</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Numero de téléphone</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit_nom"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Prenom"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_age"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Sexe"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_LR"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_tel"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QToolBox" name="toolBox_2">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>40</y>
         <width>321</width>
         <height>131</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>321</width>
          <height>77</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Spécialité du medecin</string>
        </attribute>
        <widget class="QLineEdit" name="lineEdit_s_med">
         <property name="geometry">
          <rect>
           <x>62</x>
           <y>19</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="g_specialites">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Gérer les spécialités</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>-4</x>
           <y>0</y>
           <width>331</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../Bureau/projet_image/mwd.png</pixmap>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>321</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>label_20</zorder>
        <zorder>label_19</zorder>
        <zorder>lineEdit_s_med</zorder>
        <zorder>g_specialites</zorder>
       </widget>
       <widget class="QWidget" name="page_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>321</width>
          <height>77</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Description</string>
        </attribute>
        <widget class="QLineEdit" name="lineEdit_Desc">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>321</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>-4</x>
           <y>-8</y>
           <width>331</width>
           <height>91</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../Bureau/projet_image/md.png</pixmap>
         </property>
        </widget>
        <zorder>label_21</zorder>
        <zorder>lineEdit_Desc</zorder>
       </widget>
      </widget>
      <widget class="QPushButton" name="Ajouter_patient">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>200</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirmer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>751</width>
         <height>231</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../Bureau/projet_image/final.jpg</pixmap>
       </property>
      </widget>
      <zorder>label_12</zorder>
      <zorder>layoutWidget</zorder>
      <zorder>toolBox_2</zorder>
      <zorder>Ajouter_patient</zorder>
     </widget>
     <widget class="QWidget" name="page_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>223</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Modifier Patient</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-1</y>
         <width>281</width>
         <height>231</height>
        </rect>
       </property>
       <property name="title">
        <string>Recherche du patient</string>
       </property>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>246</width>
          <height>53</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Numero de telephone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="num_modif"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="trouver">
           <property name="text">
            <string>Trouver</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>-4</x>
          <y>22</y>
          <width>281</width>
          <height>211</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Bureau/projet_image/m1.jpg</pixmap>
        </property>
       </widget>
       <zorder>label_23</zorder>
       <zorder>layoutWidget_2</zorder>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>289</x>
         <y>-1</y>
         <width>461</width>
         <height>231</height>
        </rect>
       </property>
       <property name="title">
        <string>Modification du patient</string>
       </property>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>433</width>
          <height>151</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Nom     </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nom_new"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Prenom</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="prenom_new"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Age      </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="age_new"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Sexe    </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sexe_new"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Lieu de residance      </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lr_new"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Numero de telephone</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="num_new"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Spécialité du medecin</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="s_medecin_new"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Description                </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="description_new"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="modifier">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>-20</x>
          <y>20</y>
          <width>481</width>
          <height>221</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Bureau/projet_image/m2.jpg</pixmap>
        </property>
       </widget>
       <zorder>label_24</zorder>
       <zorder>layoutWidget_3</zorder>
       <zorder>modifier</zorder>
      </widget>
      <zorder>groupBox_3</zorder>
      <zorder>groupBox_2</zorder>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>223</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Supprimer Patient</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>70</y>
         <width>262</width>
         <height>53</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Numero mobile du Patient     </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Stel"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_supprimerP">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>751</width>
         <height>221</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../Bureau/projet_image/supppatient.jpg</pixmap>
       </property>
      </widget>
      <zorder>label_18</zorder>
      <zorder>layoutWidget_4</zorder>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>223</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Afficher la liste des patients</string>
      </attribute>
      <widget class="QTableView" name="tabpatient">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>751</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="push_stats">
       <property name="geometry">
        <rect>
         <x>654</x>
         <y>200</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Statistiques</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>-4</x>
         <y>180</y>
         <width>761</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../Bureau/projet_image/modif.jpg</pixmap>
       </property>
      </widget>
      <zorder>label_22</zorder>
      <zorder>tabpatient</zorder>
      <zorder>push_stats</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
